pkg/image/registry/imagestream/strategy_test.go

```diff
@@ -696,7 +696,7 @@ func TestTagsChanged(t *testing.T) {
 			},
 		},
 		"object reference to image stream image in same stream": {
-			stream: "registry:5000/ns/stream",
+			stream: "internalregistry:5000/ns/stream",
```

Comment:
I had to open up the full file to see that this is the test case I was looking for (not a bad thing, just hard to figure out from just the diff).